addons/rating/views/rating_templates.xml

```diff
@@ -20,6 +20,7 @@
         <template id="rating_external_page_submit" name="Rate our Services">
             <t t-call="web.frontend_layout">
                 <div class="container mb-5 mt-4 o_rating_page_submit">
+                    <div class="oe_structure" id="oe_structure_rate_service_header"/>
```

Comment:
rate_service ? isn't something like "oe_structure_rate_submit_header" a bit more inlined with the template name ?